README.md

```diff
@@ -1001,6 +1001,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [Boostnote](https://boostnote.io/) - The note-taking app for programmers that focuses on markdown, snippets, and customizability. ([Source Code](https://github.com/BoostIO/Boostnote)) `GPL-3.0` `JavaScript`
 - [CodiMD](https://demo.codimd.org/) - Realtime collaborative markdown notes on all platforms, formerly HackMD CE. ([Source Code](https://github.com/hackmdio/codimd)) `AGPL-3.0` `Node.js`
 - [dillinger](http://dillinger.io/) - The last Markdown editor, ever. ([Source Code](https://github.com/joemccann/dillinger)) `MIT` `Nodejs`
+- [Dnote](https://dnote.io) - A simple and encrypted notebook for developers. Protect your privacy and stay focused. ([Source Code ](https://github.com/dnote/dnote)) `AGPL-3.0` `Go`
```

Comment:
Extra space after  `Source code`

Reply:
@nodiscc thanks for catching it. Fixed in https://github.com/Kickball/awesome-selfhosted/pull/1761/commits/77bd42fba8c10a8ff4684fa9a98fa3992c7b6d8a